packages/terra-divider/src/_SectionDivider.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import PropTypes from 'prop-types';
+import styles from './Divider.module.scss';
+
+const propTypes = {
+  /**
+   * String to be displayed inline with the divider.
+   * A level should also be used whenever using the text.
+   */
+  text: PropTypes.string.isRequired,
+  /**
+   * Sets the heading level to One of `3`, `4`, `5`, `6`. This helps screen readers to announce appropriate heading levels.
+   * Changing `level` will not visually change the style of the content.
+   */
+  level: PropTypes.oneOf([3, 4, 5, 6]).isRequired,
```

Comment:
should we keep the default value for this prop ?

Reply:
consumers will typically never set the level to a level 1 or 2, as there is always going to be at least a 1 or 2 somewhere on the page, so a 3,4,5, or 6 will be most common.

Reply:
Although levels will typically not be 1 or 2, we can't assume they will never need them to be.
```suggestion
  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]).isRequired,
```